pkg/wwan/usr/bin/wwan-init.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
-# set -x
+set -x
```

Comment:
This is generating lots of logs at least on devices without a wwan interface. We should revert.